### ISSUE TITLE
ignore key in pyrightconfig.json has no effect

### ISSUE THREAD

**Issue by Emerentius** [microsoft/pyright#265]
**Describe the bug**
I've added the `ignore` key to the `pyrightconfig.json` with paths to files and/or subdirectories with and without wildcards and nothing has any effect on the displayed errors.

**To Reproduce**
In a new directory, create file `foo.py` with this content

```python
# provoke type error
a: int = None
```

and this `pyrightconfig.json` next to it
```json
{
    "ignore": ["foo.py"],
}
```

Open VS Code in the directory and check the reported errors.

**Expected behavior**
No more warnings and errors from the `ignore`d file.

**Screenshots or Code**
![ignored_ignore](https://user-images.githubusercontent.com/9992929/65593394-aea75f80-df90-11e9-9cfc-2df058090a1e.PNG)

**VS Code extension or command-line**
I'm using VS Code 1.38.1 with the pyright extension version 1.0.65 on Windows 10.

**Comment by erictraut** [comment]
All paths within the "ignore" list are assumed to be relative to the directory that contains the pyrightconfig.json file. I can't tell from the screen shot, but I'm guessing that's the problem. Is "foo.py" in the same directory as "pyrightconfig.json" in your example above? If you want it to exclude all files called foo.py anywhere in your project's directory hierarchy, you can specify the path "**/foo.py".

**Comment by Emerentius** [comment]
Yes, it's in the same directory.

**Comment by erictraut** [comment]
Ah, must be a Windows-specific path issue. I'll dig into it more.

**Comment by erictraut** [comment]
As expected, this was a Windows-specific path issue ("\" vs "/"). The fix will be in the next release of pyright. Thanks again for reporting the issue.

**Comment by erictraut** [comment]
This is now fixed in 1.0.66, which I just published.
